packages/migrate/Migration.js

```diff
@@ -23,6 +23,14 @@ class Migration {
     this.config = config || {};
   }
 
+  static getNumber(fileName) {
+    const match = fileName.match(/\d+/);
+    if (!match.length) {
```

Comment:
[String.prototype.match](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#return_value) returns null if no match is found.

Reply:
right, thanks, i'll fix it

Reply:
done :D